test/Conversion/ImportVerilog/classes.mlir

```diff
@@ -0,0 +1,53 @@
+// RUN: circt-opt %s --moore-create-virtual-dispatch-tables --verify-diagnostics | FileCheck %s
+
+module {
```

Comment:
Maybe drop the `module { ... }` wrapper.

Reply:
Done!